sections/dashboard/FuturesMarketsTable/FuturesMarketsTable.tsx

```diff
@@ -30,60 +30,70 @@ const FuturesMarketsTable: FC<FuturesMarketsTableProps> = ({
-		const futuresVolume: FuturesVolumes = futuresVolumeQuery?.data ?? ({} as FuturesVolumes);
+	let data = useMemo(
+		() => {
+			const dailyPriceChanges = dailyPriceChangesQuery?.data ?? [];
+			const futuresVolume: FuturesVolumes = futuresVolumeQuery?.data ?? ({} as FuturesVolumes);
 
-		return futuresMarkets.map((market: FuturesMarket, i: number) => {
-			const description = getSynthDescription(market.asset);
-			const volume = futuresVolume[market.assetHex];
-			const pastPrice = dailyPriceChanges.find((price: Price) => price.synth === market.asset);
+			return futuresMarkets.map((market: FuturesMarket, i: number) => {
+				const description = getSynthDescription(market.asset);
+				const volume = futuresVolume[market.assetHex];
+				const pastPrice = dailyPriceChanges.find((price: Price) => price.synth === market.asset);
 
-			return {
-				asset: market.asset,
-				market: (market.asset[0] === 's' ? market.asset.slice(1) : market.asset) + '-PERP',
-				synth: synthsMap[market.asset],
-				description: description,
-				price: market.price.toNumber(),
-				volume: volume?.toNumber() || 0,
-				pastPrice: pastPrice?.price || '-',
-				priceChange: (market.price.toNumber() - pastPrice?.price) / market.price.toNumber() || '-',
-				fundingRate: market.currentFundingRate.toNumber(),
-				openInterest: market.marketSize.mul(market.price).toNumber(),
-				openInterestNative: market.marketSize.toNumber(),
-				longInterest: market.marketSize
-					.add(market.marketSkew)
-					.div('2')
-					.abs()
-					.mul(market.price)
-					.toNumber(),
-				shortInterest: market.marketSize
-					.sub(market.marketSkew)
-					.div('2')
-					.abs()
-					.mul(market.price)
-					.toNumber(),
-				marketSkew: market.marketSkew,
-			};
-		});
-	}, [
-		synthsMap,
-		futuresMarkets,
-		dailyPriceChangesQuery?.data,
-		futuresVolumeQuery?.data,
-		getSynthDescription,
-	]);
+				return {
+					route: parseSynthKey(market.asset),
+					asset: market.asset,
+					market: (market.asset[0] === 's' ? market.asset.slice(1) : market.asset) + '-PERP',
+					synth: synthsMap[market.asset],
+					description: description,
+					price: market.price.toNumber(),
+					volume: volume?.toNumber() || 0,
+					pastPrice: pastPrice?.price || '-',
+					priceChange:
+						(market.price.toNumber() - pastPrice?.price) / market.price.toNumber() || '-',
+					fundingRate: market.currentFundingRate.toNumber(),
+					openInterest: market.marketSize.mul(market.price).toNumber(),
+					openInterestNative: market.marketSize.toNumber(),
+					longInterest: market.marketSize
+						.add(market.marketSkew)
+						.div('2')
+						.abs()
+						.mul(market.price)
+						.toNumber(),
+					shortInterest: market.marketSize
+						.sub(market.marketSkew)
+						.div('2')
+						.abs()
+						.mul(market.price)
+						.toNumber(),
+					marketSkew: market.marketSkew,
+				};
+			});
+		},
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
this is the other main change here - the formatting changes across `data` are a result of ignoring the `exhaustive-deps`